testing/tests/DevExpress.viz.core/axisDrawing.tests.js

```diff
@@ -2120,8 +2120,6 @@ QUnit.test("Without text, all variations. Do not draw labels", function(assert)
                         return undefined;
                     case 3:
                         return "";
-                    case 1:
-                        return "      ";
                 }
```

Comment:
I think there was a typo. We need to leave this case
should be 
`
case 4:	
                        return "      ";
`

Reply:
I changed second "case 1" to "case 4".